scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Minor: no need to point out what the main parameters are since they are in the fuction call anyway.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>